Common/src/main/java/com/tabnineCommon/chat/actions/toolWindowActions/ChatSettingsAction.kt

```diff
@@ -0,0 +1,21 @@
+package com.tabnine.chat.actions.toolWindowActions
+
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.ui.IconManager
+import com.tabnineCommon.chat.ChatBrowser
+import com.tabnineCommon.chat.actions.TabnineActionRequest
+import com.tabnineCommon.chat.actions.TabnineChatAction
+import com.tabnineCommon.chat.actions.toolWindowActions.MoveToViewPayload
+
+class ChatSettingsAction(browser: ChatBrowser) :
+    TabnineChatAction(
+        browser,
+        "Settings",
+        "Settings",
+        IconManager.getInstance().getIcon("/icons/settings.svg", ChatSettingsAction::class.java)
```

Comment:
use `AllIcons.Ide.Notification.Gear`
![image](https://github.com/codota/tabnine-intellij/assets/67855609/487bd1e0-a7e1-4b89-9d43-f71a133fa799)

Reply:
There is no gear in older versions...